### PR TITLE
Doc: add a "Cloud Administration" page.

### PR DESCRIPTION
**Pull request by concretevitamin**
Prompted by https://github.com/skypilot-org/skypilot/issues/1883. 

Tested (run the relevant ones):

- [x] Any manual or new tests for this PR (please specify below)
  - rendered manually

### CODE REVIEW

**Comment by concretevitamin** [comment]
We can also call this "Cloud-specific Features" or  "Cloud-specific Guides".

File: docs/source/reference/cloud-administration.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+To use it, ensure that your machine `has AWS CLI V2 installed <https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html>`_ (by default, ``pip install skypilot[aws]`` installs V1; V2 cannot be installed via pip).
+
+You can use the following to check version:
+
+.. code-block:: console
+
+    $ aws --version
+
+
+Then, after the usual ``aws configure sso`` and ``aws sso login --profile <profile_name>`` commands, SkyPilot will work as usual.
```
Comment (Michaelvll) [inline docs/source/reference/cloud-administration.rst:35]: Are we sure that SkyPilot will work with v2 as the default `aws` command?

Currently, what I did for SSO was to create an `awsv2` command that is only used for `awsv2 configure sso` and `awsv2 sso login`, while let the SkyPilot uses the `v1`.

**Review by Michaelvll — COMMENTED** [review]
Thanks for adding the doc page @concretevitamin! Left a question for it.

File: docs/source/reference/cloud-administration.rst (hunk 1 above)
Comment (Taytay) [reply docs/source/reference/cloud-administration.rst:35]: Working like a champ for me. I admit I'm using a subset of features though. Is there a list of CLI-based commands that Sky runs?

Comment (concretevitamin) [reply docs/source/reference/cloud-administration.rst:35]: Tested all of the above with `aws` aliased to v2.

**Review by Michaelvll — APPROVED** [review]
Thanks for testing them out @concretevitamin! LGTM.

**Pull request merged**